test/unit/event-handler-wrap-error-handler-test.ts

```diff
@@ -1,17 +1,16 @@
 import simple from "simple-mock";
-import { test } from "tap";
-import { wrapErrorHandler } from "../../pkg/dist-src/event-handler/wrap-error-handler";
+import { wrapErrorHandler } from "../../src/event-handler/wrap-error-handler";
 
 test("error thrown in error handler", (t) => {
   t.plan(2);
```

Comment:
Not sure what this is used for. Is it safe to remove?

Reply:
I don't think there is something comparable in Jest. This let's you make sure that all assertions are actually run. For example if you have assertions in  callback and the callback is not called at all, then the tests passes although it should not. So when removing the `t.plan` lines, make sure to check that all assertions are run

Reply:
Maybe this does something similar?
https://jestjs.io/docs/en/expect#expectassertionsnumber

Reply:
oh yes, great find, thanks! Let's use that instead